File: app/src/main/java/com/dhsdevelopments/potato/clientapi/ClearNotificationsResult.java
package com.dhsdevelopments.potato.clientapi;

import com.google.gson.annotations.SerializedName;

public class ClearNotificationsResult
{
    @SerializedName( "result" )
    public String result;

    @Override
    public String toString() {
        return "ClearNotificationsResult[" +
                       "result='" + result + '\'' +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/ClearNotificationsResult.kt
package com.dhsdevelopments.potato.clientapi

import com.google.gson.annotations.SerializedName

class ClearNotificationsResult {
    @SerializedName("result")
    lateinit var result: String

    override fun toString(): String {
        return "ClearNotificationsResult[result='$result']"
    }
}
